README.md

```diff
@@ -12,3 +12,45 @@ Xcode 12.0
 2. [Project Configurations](https://github.com/nimblehq/ios-templates/wiki/Project-configurations)
 3. [Why having project's dependencies](https://github.com/nimblehq/ios-templates/wiki/Why-having-project%27s-dependencies)
 4. [Github Actions](https://github.com/nimblehq/ios-templates/wiki/Github-Actions-Templates)
+
+</br>
```

Comment:
I think we don't need the tag `</br>` here, please check and remove all tag `</br>` in this document